common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -278,7 +278,7 @@ public double getDouble(String key) {
     throw new RuntimeException("Invalid configuration key " + key + ".");
   }
 
-  private float getFloat(String key, final float defaultValue) {
+  private float getFloat(String key) {
```

Comment:
it's not being used anywhere and I turned it private to make sure it's not used until we fix it (drop the default value)